scripts/servers.py

```diff
@@ -520,12 +518,13 @@ def managed_webdriver_server(chrome_version=None):
                 'https://chromedriver.chromium.org/downloads/version-selection'
                 % chrome_command.replace(' ', r'\ '))
 
-        installed_version_parts = ''.join(re.findall(r'[0-9\.]', output))
-        installed_version = '.'.join(installed_version_parts.split('.')[:-1])
+        installed_version_parts = b''.join(re.findall(rb'[0-9.]', output))
+        installed_version = '.'.join(
+            installed_version_parts.decode(encoding='utf-8').split('.')[:-1])
```

Comment:
I would suggest having something in the coding style guide or similar (or maybe a separate wiki page) explaining string processing in Oppia -- i.e. when to use UNICODE, encode, decode, etc. in Python 3, especially as it relates to the Oppia codebase. (It isn't intuitive to me yet why we need to do the decoding here.)

Reply:
Done.